pageserver/src/http/openapi_spec.yml

```diff
@@ -1274,6 +1367,31 @@ components:
             tenant_id:
               type: string
               format: hex
+    TenantLocationConfigRequest:
```

Comment:
Is this supposed to be referred from https://github.com/neondatabase/neon/pull/5860/files#r1409124675?

If so the names do not match.

Reply:
Good spot.  Committed the suggestion above that fixes it.